app/models/case_contact.rb

```diff
@@ -40,16 +40,28 @@ class CaseContact < ApplicationRecord
   after_save_commit ::CaseContactMetadataCallback.new
 
   # Corresponds to the steps in the controller, so validations for certain columns can happen at those steps.
-  # These steps must be listed in order and have an html template in case_contacts/form.
+  # These steps must be listed in order, have an html template in case_contacts/form, & be listed in the status enum
   FORM_STEPS = %i[details notes expenses].freeze
-  enum status: (%w[started active] + FORM_STEPS.map(&:to_s)).zip((%w[started active] + FORM_STEPS.map(&:to_s))).to_h
```

Comment:
This was written this way to allow adding wizard form steps and have them automatically incorporated into the enum. I don't think that is worth it being so hard to read. I would rather define explicitly. It evaluates to:
```ruby
status: {
  "started"=>"started", 
  "active"=>"active", 
  "details"=>"details", 
  "notes"=>"notes", 
  "expenses"=>"expenses"
}
```